src/storage/v2/vertex_accessor.cpp

```diff
@@ -158,6 +161,8 @@ Result<bool> VertexAccessor::HasLabel(LabelId label, View view) const {
   return has_label;
 }
 
+LabelId VertexAccessor::PrimaryLabel() const noexcept { return vertex_->primary_label; }
```

Comment:
After checking the PR, two questions here:
1. Do we need to expose the priamry label of the vertex? I think until we don't use it, we can emit this function.
2. I think I forgot about the existing and deleted stuff. So if we want to expose the primary label, then maybe it worth to check whether the vertex exists/deleted.

Reply:
1. This is used throughout tests to confirm that the correct `PrimaryLabel` is set
2. I agree I will add that logic for fetching `PrimaryLabel`